extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -221,40 +221,45 @@ export class ResourceTypePickerDialog extends DialogBase {
 				if (this.toolRefreshTimestamp !== currentRefreshTimestamp) {
 					return;
 				}
-				let installationNeeded = false;
 				let minVersionCheckFailed = false;
-				const messages: string[] = [];
+				let toolsToAutoInstall: ITool[] = [];
+				let messages: string[] = [];
 				this._toolsTable.data = toolRequirements.map(toolRequirement => {
 					const tool = this.toolsService.getToolByName(toolRequirement.name)!;
 					// subscribe to onUpdateData event of the tool.
 					this._toDispose.push(tool.onDidUpdateData((t: ITool) => {
 						this.updateToolsDisplayTableData(t);
 					}));
-					if (tool.isNotInstalled && !tool.autoInstallSupported) {
-						messages.push(localize('deploymentDialog.ToolInformation', "'{0}' was not discovered and automated installation is not supported. Kindly install it manually or if installed make sure it is started and discoverable. Once done please restart Azure Data Studio. See [{1}] .", tool.displayName, tool.homePage));
+					if (tool.isNotInstalled) {
+						if (tool.autoInstallSupported) {
+							toolsToAutoInstall.push(tool);
+						} else {
+							messages.push(localize('deploymentDialog.ToolInformation', "'{0}' was not discovered and automated installation is not supported at the moment. Install '{0}' manually or ensure it is started and discoverable. Once done please restart Azure Data Studio. See [{1}] .", tool.displayName, tool.homePage));
```

Comment:
Run this string by a PM. I don't think "at the moment" is the best here. Something like "not currently supported" is better since moment implies that it may be available if they try again.

Reply:
See https://github.com/microsoft/azuredatastudio/issues/8638. Dinaker specifically requested this verbiage. I will still double-check with him.

Reply:
Added Dinaker to chime in.